far/editor.cpp

```diff
@@ -78,9 +78,10 @@ THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 //----------------------------------------------------------------------------
 
-static bool ReplaceMode, ReplaceAll;
+// $ 2023-01-15 MZK: Do we really want to share them between Editor instances?
+//static bool ReplaceMode, ReplaceAll;
```

Comment:
🙂 I made this comment (and the other one), instead of removing the whole thing, as a reminder to discuss the behavioral consequences and desirability of replacing global statics with instance variables. It was long before creating the PR, so there was no other place to make this note. Then I kinda forgot about it, the PR being a draft.

So, looks like we are OK with this change. I will remove the dead code for the next iteration.

Reply:
Should be ok. Even if we later discover that we want the shared state back, it would be better to make it more obvious than it was.